config/production.yaml

```diff
@@ -11,93 +11,7 @@ providers:
     delete_pcent_threshold: 1.0
 
 zones:
-  hackclub.com.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  hackclub.org.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  hackclub.io.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  hackedu.us.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  hack.af.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  bulckcah.com.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  dino.icu.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  dinosaurbbq.org.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  hack.club.:
-    sources:
-      - config
-    targets:
-      - dnsimple
-  hackclub.app.:
-    lenient: true
-    sources:
-      - config
-    targets:
-      - dnsimple
```

Comment:
It looks like there's some custom config here -- how should we handle this?

Reply:
@Muirrum can you elaborate here?